### PR TITLE
Add FrancescoBal SanDiego File

### LINKED ISSUE #40: Create a new file for FrancescoBal San Diego, CA
- [x] Create a branch
- [x] Add Commits
- [x] Open PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by FrancescoBal**
closes #40

### CODE REVIEW

File: FrancescoBal-SanDiego.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# San Diego, CA
+
+### Food Recommendations
+- Rubios
+- Monello
+- Pollo Loco
```
Comment (bhagyesh12) [inline FrancescoBal-SanDiego.md:7]: Do they have something good for vegans?

File: FrancescoBal-SanDiego.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# San Diego, CA
```
Comment (Tamem-Maaz) [inline FrancescoBal-SanDiego.md:1]: How hot is it right now?

**Comment by Tamem-Maaz** [comment]
👍 💯 🥇

**Review by bhagyesh12 — APPROVED** [review]
Assuming no vegan options, so all good!! :walking_man:

**Review by Tamem-Maaz — APPROVED** [review]
This looks great. 👍

**Pull request merged**